packages/loader/container-loader/src/utils.ts

```diff
@@ -70,6 +70,9 @@ function convertSummaryToSnapshotWithEmbeddedBlobContents(
                 treeNode.trees[key] = convertSummaryToSnapshotWithEmbeddedBlobContents(summaryObject);
                 break;
             }
+            case SummaryType.Attachment:
```

Comment:
How is this related to the rest of the change?

Reply:
This function is used during rehydration from snapshot. Since only BlobManager uses the attachment type and previously, blobs were not supported in detached containers, there wouldn't ever be attachment types in the snapshot. This is covered in the unit test added in this PR as part of container serialization/rehydration.